db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -4785,13 +4787,25 @@ declareTemporaryTableDeclaration() throws StandardException :
     TableName            tableName;
     TableElementList tableElementList;
     Object[] declareTableClauses = new Object[3];
+    Token ifTok = null;
+    Token notTok = null;
+    Token existsTok = null;
+    Token ifTok2 = null;
+    Token notTok2 = null;
+    Token existsTok2 = null;
 }
 {
-    <DECLARE> <GLOBAL> <TEMPORARY> <TABLE> tableName = qualifiedName(Limits.MAX_IDENTIFIER_LENGTH)
+    <DECLARE> <GLOBAL> <TEMPORARY> <TABLE> [ ifTok = <IF> notTok = <NOT> existsTok = <EXISTS> ] tableName = qualifiedName(Limits.MAX_IDENTIFIER_LENGTH) [ ifTok2 = <IF> notTok2 = <NOT> existsTok2 = <EXISTS> ]
```

Comment:
Is it possible to simplify this pattern a little bit? At least remove the "notTok" and "existsTok" variables, I don't think those are needed, and then have`boolean ifNotExists = ifTok != null || ifTok2 != null;`

Reply:
I'll give it a try. I'm not yet familiar with javacc grammar and got this from `CREATE SCHEMA IF NOT EXISTS ...` implementation.

Reply:
Done.